dev-python/ipympl/ipympl-0.9.7.ebuild

```
@@ -0,0 +1,39 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+
+DISTUTILS_USE_PEP517=hatchling
+inherit distutils-r1 pypi
+
+DESCRIPTION="Matplotlib extension for Jupyter"
+HOMEPAGE="https://matplotlib.org/ipympl https://github.com/matplotlib/ipympl https://pypi.org/project/ipympl"
```

```suggestion
HOMEPAGE="
	https://matplotlib.org/ipympl/
	https://github.com/matplotlib/ipympl/
	https://pypi.org/project/ipympl/
"
```

Try to keep lines below 80 chars.